swan-lake/resources/featured-scenarios/build-a-data-service-in-ballerina.md

```diff
@@ -59,222 +55,257 @@ CREATE TABLE Company.Employees (
 
-If the program runs without throwing an error, that indicates that the connection has been established successfully. This client can be defined globally and be used across all parts of the program.
+Running executable
+```
 
->**Info:** The MySQL package provides additional connection options and the ability to configure connection pool 
->properties when connecting to the database which, are not covered in this tutorial. To learn more about this, 
->see [`mysql:Client`](https://lib.ballerina.io/ballerinax/mysql/latest#Client).
+>**Info:** The MySQL package provides additional connection options and the ability to configure connection pool properties when connecting to the database, which are not covered in this tutorial. To learn more about this, see [`mysql:Client`](https://lib.ballerina.io/ballerinax/mysql/latest#Client).
 
 ### Execute the queries
 
 The `mysql:Client` provides two primary remote methods for performing queries.
 
 1. `query()` - Executes an SQL query and returns the results (rows) from the query. 
-   The `queryRow()` method is a variation of this method, which returns at most a single row from the result.
-
-2. `execute()` - Executes an SQL query and returns only the metadata of the execution.
-
-To use the `query()`, `queryRow()`, and `execute()` methods, which can perform basic CRUD operations against the MySQL database, add the code below to the `main.bal` file. 
-
-```ballerina
-isolated function addEmployee(Employee emp) returns int|error {
-    sql:ExecutionResult result = check dbClient->execute(`
-        INSERT INTO Employees (employee_id, first_name, last_name, email, phone,
-                               hire_date, manager_id, job_title)
-        VALUES (${emp.employee_id}, ${emp.first_name}, ${emp.last_name},  
-                ${emp.email}, ${emp.phone}, ${emp.hire_date}, ${emp.manager_id},
-                ${emp.job_title})
-    `);
-    int|string? lastInsertId = result.lastInsertId;
-    if lastInsertId is int {
-        return lastInsertId;
-    } else {
-        return error("Unable to obtain last insert ID");
+   The `queryRow()` method is a variation of this method, which returns, at most, a single row from the result.
+
+2. `execute()` - Executes an SQL query and returns only the execution metadata.
+
+#### Create the functions
+
+You need to create `isolated` functions to use the `query()`, `queryRow()`, and `execute()` methods, which can perform basic CRUD operations against the MySQL database. 
+
+Follow the steps below to create the functions.
+
+1. Add the code below to import the [`sql`](https://lib.ballerina.io/ballerina/sql/latest) package, which is used in the logic of the functions.
+
+    ```ballerina
+    import ballerina/sql;
+    ```
+2. Create the first function, as shown below.
+
+    ![Create isolated function](/learn/images/featured-scenarios/build-a-data-service-in-ballerina/create-isolated-function.gif)
+
+3. Add the code below to the body of the function.
+
```

Comment:
I'm still not sure if we need the UI approach for this.

Reply:
For resources/services it makes more sense with the dropdowns, options, etc. Not as much for normal functions.

WDYT? @praneesha @anupama-pathirage

Reply:
IMHO, it would be better to show the tooling capabilities as much as possible. We can show how to create the function with the name, parameters, and return types via tooling.

Reply:
I am agreeing with @MaryamZi here, no point in using UI features for normal functions. For resource functions it makes sense as we can do more things like adding payloads, define custom error types with specific HTTP error codes etc (which is more common in resource functions)

Reply:
Removed the tooling approach for creating the functions.